### PR TITLE
feat(ui): Use PlatformList as ProjectBadge Avatar

### PR DESCRIPTION
**Pull request by billyvg**
* Change Project breadcrumb to display icons
* Change ProjectSelector to use IdBadge
* Change ProjectCard to use IdBadge
* Show project icons for active project in header (internal only)

### CODE REVIEW

File: src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx [hunk 1]
```diff
@@ -277,17 +268,23 @@ const ProjectSelector = createReactClass({
   },
 
   getLinkNode(team, project) {
-    let org = this.props.organization;
+    let {organization} = this.props;
     let label = this.getProjectLabel(team, project);
 
-    if (!this.context.location) {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx]: our `<Link>` component does this logic already.

File: src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx [hunk 2]
```diff
@@ -1,21 +1,18 @@
+import {browserHistory} from 'react-router';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx]: Sorry my auto import plugin sorts imports alphabetically + groups them

File: src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx [hunk 3]
```diff
@@ -223,14 +214,15 @@ const ProjectSelector = createReactClass({
 
     return (
       <MenuItem {...menuItemProps}>
-        {/*ideally an icon here...*/}
-        <ProjectBadge>
-          <Flex>
-            <StyledPlatformList platforms={platforms} direction="left" />
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx]: I think the direction logic was reversed...

**Comment by billyvg** [comment]
Will update after https://github.com/getsentry/sentry/pull/8864

File: src/sentry/static/sentry/app/components/platformList.jsx [hunk 4]
```diff
@@ -5,61 +5,83 @@ import styled from 'react-emotion';
...
+  };
 
-  getIcons(platforms) {
+  getIcons = platforms => {
     return platforms
       .slice()
       .reverse()
       .map(this.getIcon);
-  }
+  };
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/platformList.jsx]: Dunno if prettier would change it, but you could just write this (I think) as

```js
getIcons = platforms =>
  platforms
    .slice()
    .reverse()
    .map(this.getIcon);
```

File: src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx]: don't be sorry for this.

**Comment by EvanPurkhiser** [comment]
Added WIP tag since it needs updated

**Comment by billyvg** [comment]
@EvanPurkhiser this could use a review now

**Review by evanpurkhiser — APPROVED** [review]
lgtm

**Pull request merged**